resolver_service/views.py

```diff
@@ -1,33 +1,33 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-import sys
-import os
-
-PROJECT_HOME = os.path.abspath(os.path.join(os.path.dirname(__file__), '.'))
-sys.path.append(PROJECT_HOME)
-
-from flask import Response
+from flask import request, Blueprint
 from flask import current_app
 from requests.utils import quote
 
-import json
 
-from linksData import *
-from adsputils import load_config, setup_logging
+bp = Blueprint('resolver_service', __name__)
 
-logger = None
-config = {}
+        
+@advertise(scopes=[], rate_limit=[1000, 3600 * 24])
+@bp.route('/<bibcode>/<linkType>', methods=['GET'])
```

Comment:
i think linktype should be optional

renamed '/v1/resolver/bibcode' to '/bibcode' - thats because the service will be inserted into api; api gateway is responsible to route '/v1/resolver' to your service - and your service willl only see bibcode and link_type

Reply:
about the naming conventions: we should follow this https://www.python.org/dev/peps/pep-0008/

Reply:
I shall ask Alberto. I don't think there is every a time to want to combine all the links for a single bibcode.